### PR TITLE
Recursion to find the exponent

### PR DESCRIPTION
**Pull request by thearkamitra**

### CODE REVIEW

**Review by tjgurwara99 — CHANGES_REQUESTED** [review]
Please read our contribution guidelines before opening a PR.

File: recursion/exponent.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package main
```
Comment (tjgurwara99) [inline recursion/exponent.go:1]: Specifically the part where it says the functions should be self contained and not in a main package.

**Review by tjgurwara99 — CHANGES_REQUESTED** [review]

File: recursion/exponent.go (hunk 1 above)
Comment (tjgurwara99) [inline recursion/exponent.go:1]: Change the package name to something meaningful and not main

File: recursion/exponent.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+import "fmt"
+
+func exponentialrecursion(x int, p int) int {
```
Comment (tjgurwara99) [inline recursion/exponent.go:5]: Export this function and write a test for this

**Review by tjgurwara99 — CHANGES_REQUESTED** [review]

**Review by siriak — APPROVED** [review]
Agree with previous comments, but apart from that looks good

**Pull request closed without merging**